x-pack/plugins/ml/public/application/routing/routes/datavisualizer/data_compare.tsx

```diff
@@ -0,0 +1,64 @@
+/*
```

Comment:
Nit: suggest to rename this file to `data_comparison.tsx` to be in line with the feature name.